pyrometheus/codegen/python.py

```diff
@@ -135,6 +135,8 @@ class Thermochemistry:
     .. automethod:: get_mixture_specific_heat_cv_mass
     .. automethod:: get_mixture_enthalpy_mass
     .. automethod:: get_mixture_internal_energy_mass
+    .. automethod:: get_specific_heat_ratio
+    .. automethod:: get_soundspeed
```

Comment:
`soundspeed` is not a word. Rename to `get_sound_speed` or `get_speed_of_sound`?